.pep8speaks.yml

```diff
@@ -4,7 +4,6 @@ pycodestyle:
     - setup.py
     - ez_setup.py
     - ah_bootstrap.py
```

Comment:
Remove theses.

Reply:
Also if these appear anywhere else too.

Reply:
even setup.py is to be removed from everywhere?

Reply:
Doing it

Reply:
`ez_setup.py` is an old file. It is not `setup.py`.